packages/gx-agent/src/utils/vc-extraction.ts

```diff
@@ -36,3 +36,42 @@ export function getIssuerString(vc: VerifiableCredential): string {
 export function isString(value: unknown): boolean {
   return typeof value === 'string' || Object.prototype.toString.call(value) === '[object String]'
 }
+
+export function getVcType(verifiableCredential: VerifiableCredential): string {
+  const sdTypes = verifiableCredential.type as string[]
```

Comment:
Array check and conversion to array to be sure. Could theoretically only be a string

Reply:
added a util function to make this a uniform type (string[]) in our operations